src/Sulu/Bundle/AdminBundle/Resources/js/stores/MultiSelectionStore/MultiSelectionStore.js

```diff
@@ -42,7 +42,7 @@ export default class MultiSelectionStore {
         this.loading = loading;
     }
 
-    @action loadItems = (itemIds: ?Array<string | number>) => {
+    @action loadItems(itemIds: ?Array<T>) {
```

Comment:
need to use a real function here because of https://github.com/facebook/flow/issues/6978